src/services/gas-price/gas-price.service.ts

```diff
@@ -57,8 +57,8 @@ export default class GasPriceService {
       if (gasPrice != null) {
         if (
           ethereumTxType.value === EthereumTxType.EIP1559 &&
-          options.maxFeePerGas != null &&
-          options.maxPriorityFeePerGas != null &&
+          gasPrice.maxFeePerGas != null &&
+          gasPrice.maxPriorityFeePerGas != null &&
```

Comment:
sneaky bug - the reason why it didn't work is because the result was on the `gasPrice` object instead of `options`.